lstchain/reco/dl0_to_dl1.py

```diff
@@ -55,20 +55,23 @@
                        }
 
 
-def get_dl1(calibrated_event, telescope_id):
+def get_dl1(calibrated_event, telescope_id, dl1_container=None):
```

Comment:
do we need to initialize `dl1_container` to a None value?

Reply:
It allows using the function without initializing a DL1ParametersContainer before